### PR TITLE
audit logs for free trial, plus test suite log cleanup

### PR DESCRIPTION
**Pull request by davidangb**
no jira. Adds audit logs for important free trial transitions, plus some log silencing in tests.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by davidangb** [comment]
jenkins retest

**Comment by davidangb** [comment]
jenkins retest

File: src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala [hunk 1]
```diff
@@ -254,9 +263,13 @@ final class TrialService
       } else {
         // 2. Add the user as Owner to the assigned Billing Project
         rawlsDAO.addUserToBillingProject(projectId, ProjectRoles.Owner, userInfo.userEmail)(userToken = saToken) flatMap { _ =>
+          logger.info(s"[trialaudit] added user ${userInfo.userEmail} (${userInfo.id}) to project $projectId")
           // 3. Update the user's Thurloe profile to indicate Enrolled status
-          thurloeDao.saveTrialStatus(userInfo.id, userInfo, enrolledStatusFromStatus(status)) flatMap {
-            case Success(_) => Future(RequestComplete(NoContent)) // <- SUCCESS!
+          val updatedTrialStatus = enrolledStatusFromStatus(status)
```
Comment (kyuksel) [inline src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala:268]: helpful!

**Review by kyuksel — COMMENTED** [review]
Logging level seems to be set to `WARN` (and higher). Won't the `INFO` loggings be ignored?

**Review by kyuksel — APPROVED** [review]

**Comment by davidangb** [comment]
about the default loglevel: loglevel is set to INFO for our code, and WARN for third-party code:
https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/resources/logback.xml#L22

**Pull request merged**